config/routes.rb

```diff
@@ -24,6 +24,21 @@
         end
       end
 
+      resources :apps, only: [], param: :name do
+        resources :versions, only: [], param: :number, constraints: { number: /[\d\.]+/ } do
+          resources :builds, only: [], param: :timestamp do
+            resources :visitors, only: [] do
+              resource :config, only: :show, controller: 'app_visitor_configs'
+            end
+            resources :identifier_types, only: [], param: :name do
+              resources :identifiers, only: [], param: :value do
+                resource :visitor_config, only: :show, controller: 'app_identifier_visitor_configs'
```

Comment:
and this one looks like:

`/api/v1/apps/:app_name/versions/:version_number/builds/:build_timestamp/identifier_types/:identifier_type_name/identifiers/:identifier_value/visitor_config`

Notably, even though we've established a different format for v2 splits and assignments endpoints, to make the lift smaller for mobile, I've built these as v1-style controllers.

Reply:
You'll definitely want to make sure your URL-interpolated values are URLencoded properly - especially the build timestamp.

Reply:
by `v1-style` do you mean, within the `v1` namespace? if so, okay!

Reply:
it's in the v1 namespace, but it also conforms to the way the v1 visitors controller renders, not the way the v2 visitors controller renders, which nobody uses yet.